tests/test_taskipy.py

```diff
@@ -247,6 +247,20 @@ def test_handling_sigint_according_to_subprocess_if_it_does_not_handle_it_gracef
 
         self.assertEqual(exit_code, 130)
 
+    def test_handling_sigterm_according_to_subprocess_if_it_does_handle_it_gracefully(self):
```

Comment:
as the behavior is now "all signals are propagated to subprocess" we can rename this test to `test_signals_should_be_sent_to_subprocess`. The test itself is ok, it's just that the name should indicate that this is the expected behavior, and not just sigterm.